.github/workflows/testing.yml

```diff
@@ -0,0 +1,174 @@
+name: testing
+
+on:
+  push:
+  pull_request:
+  pull_request_target:
+    types: [labeled]
+  workflow_dispatch:
+
+jobs:
+  run-tests-ee:
+    # Does not run on pull requests from forks and on forks by default.
+    # Tests from forks will run only when the pull request is labeled with
+    # `full-ci`. To avoid security problems, the label must be reset manually
+    # for every run.
+    #
+    # We need to use `pull_request_target` because it has access to base
+    # repository secrets unlike `pull_request`.
+    if: |
+      github.repository == 'tarantool/go-tlsdialer' &&
+      (github.event_name == 'push' ||
+        (github.event_name == 'pull_request_target' &&
+          github.event.pull_request.head.repo.full_name != github.repository &&
+          github.event.label.name == 'full-ci')) ||
+      github.event_name == 'workflow_dispatch'
+
+    runs-on: ubuntu-latest
+
+    env:
+      COVERAGE_FILE: 'coverage.out'
+
+    strategy:
+      fail-fast: false
+      matrix:
+        golang:
+          - '1.20'
```

Comment:
Doesn't go-tarantool still supports 1.13?

Reply:
`go-tarantool v2` will also update go version to 1.20.